src/components/Picker/Picker.scss

```diff
@@ -78,10 +78,11 @@ $stacking-order: (
   margin: -5px 0 0;
   padding: 0;
   background: color(white);
-  box-shadow: 0 2px 5px 0 rgba(0, 0, 0, 0.25);
+  box-shadow: 0 5px 5px 0 rgba(0, 0, 0, 0.25);
```

Comment:
@Farhan-emgage please use/update the shadow map: https://github.com/emgage/engage-ui/blob/master/src/styles/foundation/shadows.scss

Reply:
@weegrom 

ok i will make update